auth/api/src/test/java/edu/iu/auth/oidc/IuOpenIdProviderTest.java

```diff
@@ -51,11 +49,10 @@ public void testUsesSpiFactory() {
 		try (final var mockSpiFactory = mockStatic(IuAuthSpiFactory.class)) {
```

Comment:
I couldn't leave this comment on the name of the method, but I'm just curious what everyone thinks about the naming conventions for the test methods. I think this name explains what we expect the method under test to do, but the name doesn't relate directly to the method we're testing. I thought it might just be called `testFrom`.